<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>北京公交查询系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Login">
      <attribute name="title">
       <string>登录</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;宋体&quot;;
font: 10pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_Name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>密 码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_Code"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_Logout">
         <property name="text">
          <string>注销</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_Login">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_Quit">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Manager">
      <attribute name="title">
       <string>信息管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;宋体&quot;;
text-decoration: underline;
font: 9pt &quot;宋体&quot;;
font: 12pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>添加一条线路信息：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>线路名：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_AddBus"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>时间：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_AddTime"/>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="label_9">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>站点数目：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>所有站点（以空格分开）：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="8">
        <widget class="QPlainTextEdit" name="plainTextEdit_AddBus"/>
       </item>
       <item row="3" column="8">
        <widget class="QPushButton" name="pushButton_Add">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;宋体&quot;;
text-decoration: underline;
font: 12pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>删除一条线路信息：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>线路名：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_DeleteBus"/>
       </item>
       <item row="5" column="8">
        <widget class="QPushButton" name="pushButton_DeleteBus">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;宋体&quot;;
text-decoration: underline;
font: 12pt &quot;宋体&quot;;</string>
         </property>
         <property name="text">
          <string>更新一条线路信息：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>线路名：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_UpdateBus"/>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>时间：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLineEdit" name="lineEdit_UpdateTime"/>
       </item>
       <item row="7" column="7">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>站点数目：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="8">
        <widget class="QSpinBox" name="spinBox_UpdateStationNum"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>所有站点（以空格分开）：</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="8">
        <widget class="QPlainTextEdit" name="plainTextEdit_UpdateStation"/>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_25">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>票价：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>票价：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_AddPrice">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="5" colspan="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_UpdatePrice">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="9" column="8">
        <widget class="QPushButton" name="pushButton_Update">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QSpinBox" name="spinBox_StationNum">
         <property name="minimum">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Search">
      <attribute name="title">
       <string>查询</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="textBrowser_AllBus"/>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>2、线路</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_OneBus"/>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>  的所有信息;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_OneBus">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QTextBrowser" name="textBrowser_OneBus"/>
       </item>
       <item row="4" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>3、经过站点：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Station"/>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>      的所有线路</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_StationQuery">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QTextBrowser" name="textBrowser_SomeBus"/>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1、所有公交路线：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>    (按照起点排序)    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_QueryAll">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Transfer">
      <attribute name="title">
       <string>换乘查询</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;宋体&quot;;
text-decoration: underline;</string>
         </property>
         <property name="text">
          <string>换乘查询：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>21</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>起点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Start">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>21</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>终点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_End">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>21</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="_2">
         <item>
          <widget class="QRadioButton" name="radioButton_Unconditional">
           <property name="text">
            <string>无条件</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButton_ChangeLess">
             <property name="text">
              <string>换乘次数少于等于</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_ChangeNum">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>199</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_DistanceLess">
           <property name="text">
            <string>距离最短</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_PriceLess">
           <property name="text">
            <string>票价最少</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_BusChange">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QTextBrowser" name="textBrowser_BusChange"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>相关信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>by 赵帅江 赵帅兵</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
